### PR TITLE
Update README and source code comments

### PR DESCRIPTION
**Pull request by yvesgurcan**
Hello 18F team! I would like to suggest this small PR to update the documentation of this library. Thank you for making this code available to the community! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,70 +15,78 @@ npm install @18f/us-federal-holidays
 To get a list of all US federal holidays in a given year, use the `allForYear` method. If no year is passed in, uses the current year.
 
 ```javascript
-var fedHolidays = require('us-federal-holidays');
-var holidays = fedHolidays.allForYear(2016);
+const fedHolidays = require('@18f/us-federal-holidays');
+
+const options = { shiftSaturdayHolidays: true, shiftSundayHolidays: true };
+const holidays = fedHolidays.allForYear(2016, options);
```
Comment (mgwalker) [inline README.md:21]: 🙌 Yes!  These should always have been `const`, thanks!

File: README.md [hunk 2]
```diff
@@ -15,70 +15,78 @@ npm install @18f/us-federal-holidays
...
 
 // Returns
 [ { name: 'New Year\'s Day',
-    date: Thu Dec 31 2015 18:00:00 GMT-0600 (CST),
+    date: 2016-01-01T00:00:00.000Z,
```
Comment (mgwalker) [inline README.md:25]: Oops, yep.  Good catch!

File: README.md [hunk 3]
```diff
@@ -15,70 +15,78 @@ npm install @18f/us-federal-holidays
...
-The options is a plain object:
+Both methods take `options` as a second argument. This argument is a plain object which accepts the following properties:
 
 ```javascript
 {
   // Whether or not holidays that fall on Saturdays should be
   // shifted to Friday observance.  Defaults to true, but if
   // you don't follow the US federal standard for observing
   // holidays on weekends, you can adjust!
+  // Default value is true.
```
Comment (mgwalker) [inline README.md:74]: The description text above says `Defaults to true` in it, but I like this where it's called out separately.  I think it makes the documentation easier to scan.

File: src/index.js [hunk 4]
```diff
@@ -103,8 +103,8 @@ function allFederalHolidaysForYear(
       const dow = holiday.date.getUTCDay();
 
       if (dow == 0 && shiftSundayHolidays) {
-        // Actual holiday falls on Sunday.  Shift
-        // the observed date forward to Monday.
+        // Actual holiday falls on Sunday.
+        // Shift the observed date forward to Monday.
```
Comment (mgwalker) [inline src/index.js:107]: These are fine, but FWIW, they may get changed again later.  The 18F style preference is to wrap at around the 80th column, so both the original and the modified are very minor violations.  This is what I get for not setting up a code formatter on this project.  😬

**Review by mgwalker — APPROVED** [review]
These are great, thanks for the updates!

File: src/index.js (hunk 4 above)
Comment (yvesgurcan) [reply src/index.js:107]: Would you be open to use ESlint? I can submit a PR and set it up for you :)

File: README.md (hunk 3 above)
Comment (yvesgurcan) [reply README.md:74]: I might open another PR to alter the comments a little bit. Here is the suggestion I have in mind:
```
  // Whether or not holidays that fall on Saturdays should be
  // shifted to Friday observance. If you don't follow the
  // US federal standard for observing holidays on weekends,
  // you can adjust by setting this value to false.
  // Default value is true.
```
What do you think?

File: src/index.js (hunk 4 above)
Comment (yvesgurcan) [reply src/index.js:107]: Also, I think I might not be understanding the wrapping-at-80 rule very well. I counted how many characters there are on both lines and it looks like it's 42 and 53. Am I missing something? Are indentations counted differently?

Comment (mgwalker) [reply src/index.js:107]: No, you're understanding correctly.  It wasn't right in the original code.  :)  I'll send up a PR in a little bit that includes eslint, prettier, and CI/CD

File: README.md (hunk 3 above)
Comment (mgwalker) [reply README.md:74]: Your way reads better.  I like it!  👍

File: src/index.js (hunk 4 above)
Comment (yvesgurcan) [reply src/index.js:107]: Great! I'm a sucker for ESLint :)

File: README.md (hunk 3 above)
Comment (yvesgurcan) [reply README.md:74]: Ok. I'll submit a PR later today.

**Pull request merged**